Doc/library/fnmatch.rst

```diff
@@ -75,7 +75,8 @@ patterns.
 
 .. function:: filter(names, pattern)
 
-   Return the subset of the list of *names* that match *pattern*. It is the same as
+   Return the subset of the list of *names* that match *pattern*.  *pattern*
+   can be a tuple of patterns.  It is the same as
```

Comment:
Please add a `versionchanged` directive for the new changes in behavior.

Reply:
Addressed your comments on Dec 13, 2020 and forgot to tell you :sweat_smile: